plugins/tailwind.ts

```diff
@@ -0,0 +1,52 @@
+import type { Plugin } from "../server.ts";
+import { type Config, tailwindcss as tailwind } from "./tailwind_deps.ts";
+import postcssPlugin, { type PostCssPluginOptions } from "./postcss.ts";
+
+/**
+ * Fresh PostCSS Tailwind Plugin
+ * @param options - {@link PostCssPluginOptions} PostCSS plugin options
+ * @param config - Tailwind configuration
+ * @returns PostCSS plugin to process Tailwind in Fresh
+ * @example
+ * ```ts
+ * import { defineConfig } from "$fresh/server.ts";
+ * import tailwindPlugin from "$fresh/plugins/tailwind.ts";
+ *
+ * export default defineConfig({
+ *   plugins: [
+ *     tailwindPlugin({
+ *       css: ["./src/styles.css"],
+ *     }, (await import("./tailwind.config.ts")).default),
+ *   ],
+ * });
+ * ```
+ */
+export default function tailwindPlugin(
+  options: PostCssPluginOptions | PostCssPluginOptions["css"],
```

Comment:
I think the additional logic to account for union types should be worth the DX gained. In this instance, I think it's fine just to have `PostCssPluginOptions`.